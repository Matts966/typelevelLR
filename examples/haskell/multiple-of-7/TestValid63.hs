
import MultipleOf7

-- 63 = 0b111111

main :: IO ()
main = print $ begin |> i |> i |> i |> i |> i |> i |> end
