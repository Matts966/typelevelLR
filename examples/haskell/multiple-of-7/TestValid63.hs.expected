RuleS1 (Rule11 (Rule31 (Rule01 (Rule11 (Rule31 Accept)))))
